client/src/components/utils/TextAreaAntd.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Form, Input, Tooltip, Icon } from "antd";
```

Comment:
while you're not using Input why create a variable for it? just destrcuture yor textarea from it
```suggestion
import { Form, Input:{ TextArea }, Tooltip, Icon } from "antd";
```